RELEASE.md

```diff
@@ -58,12 +58,15 @@ This release contains contributions from:
 * <SIMILAR TO ABOVE SECTION, BUT FOR OTHER IMPORTANT CHANGES / BUG FIXES>
 * <IF A CHANGE CLOSES A GITHUB ISSUE, IT SHOULD BE DOCUMENTED HERE>
 * <NOTES SHOULD BE GROUPED PER AREA>
+* Scipy minimize wrapper caches compiled graphs and re-uses them if called with the same arguments.
```

Comment:
Maybe mention that this can be disabled by setting the new compile_cache_size parameter?

Reply:
Done